### PR TITLE
Adds reload and edit buttons to the MaterialXNode.

### PR DESCRIPTION
**Pull request by feldstj**
Note: reload currently calls an empty command.

### CODE REVIEW

**Comment by bernardkwok** [comment]
Double checking is popen() secure in terms of it's arguments. I assume there should be some argument checker? I'm wondering why we are not using commandPort ? 

Also I'm not sure if we should hardcode LookdevX.exe as the editor. First of all it exposes a component name we have not publish, and second is is hardcoded.

**Comment by ppenenko** [comment]
I'm afraid this PR will have to be redone to adsk_contrib/dev - probably by cherry-picking to a new branch based on adsk_contrib/dev

File: source/MaterialXContrib/MaterialXNode/scripts/MaterialXNodeUtil.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from subprocess import Popen
+from distutils.spawn import find_executable
+
+def editMaterialXDocument(documentPath, element, editor):
+    editor_path = find_executable(editor)
+    if editor_path == None:
+        print ("Unable to find editor: " + editor)
+    else:
+        print "Launching editor..."
+        cmd = "system(\"start " + editor + " \\\"" + documentPath + "\\\"\");"
```
Comment (bernardkwok) [inline source/MaterialXContrib/MaterialXNode/scripts/MaterialXNodeUtil.py]: \ wont' work on Linux.

File: source/MaterialXContrib/MaterialXNode/scripts/MaterialXNodeUtil.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        mel.eval(cmd)
+
+def editMaterialXNode(nodeName):
+    documentFilePath = cmds.getAttr(nodeName + "documentFilePath")
+    elementPath = cmds.getAttr(nodeName + "elementPath")
+    editor = os.environ['MAYA_MATERIALX_EDITOR'] if 'MAYA_MATERIALX_EDITOR' in os.environ else "Notepad.exe"
```
Comment (bernardkwok) [inline source/MaterialXContrib/MaterialXNode/scripts/MaterialXNodeUtil.py]: There is no notepad on Linux or Mac so maybe make this vi on those platforms.

Comment (bernardkwok) [reply source/MaterialXContrib/MaterialXNode/scripts/MaterialXNodeUtil.py]: I think there is actually a Maya text editor preference but maybe it's easiest to make this an optionVar.

File: source/MaterialXContrib/MaterialXNode/scripts/MaterialXNodeUtil.py (hunk 1 above)
Comment (feldstj) [reply source/MaterialXContrib/MaterialXNode/scripts/MaterialXNodeUtil.py]: The '\'s are for quotes, not paths

**Pull request merged**